GeeksCoreLibrary/Core/Helpers/HttpContextHelpers.cs

```diff
@@ -328,7 +328,11 @@ public static string GetUserIpAddress(HttpContext httpContext)
         var result = GetHeaderValueAs<string>(httpContext, "CF_CONNECTING_IP"); // Cloud Flare IP address.
         if (String.IsNullOrWhiteSpace(result))
         {
-            result = GetHeaderValueAs<string>(httpContext, "X_FORWARDED_FOR");
+            result = GetHeaderValueAs<string>(httpContext, "True-Client-IP"); // Also Cloud Flare IP address.
+            }
+            if (String.IsNullOrWhiteSpace(result))
+            {
+                result = GetHeaderValueAs<string>(httpContext, "X_FORWARDED_FOR"); // TransIP and other providers use this.
```

Comment:
It looks like the indentation here got a bit messed up after the rebase. Can you fix this?

Reply:
Sorry, didn't notice this. Is fixed now.